xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VaapiEGL.cpp

```diff
@@ -187,38 +187,6 @@ bool CVaapiTexture::Map(CVaapiRenderPicture *pic)
 
       break;
     }
-    case VA_FOURCC('B','G','R','A'):
```

Comment:
keep it

Reply:
You mean 1681a9a completely? As far as I could tell, the resulting imported texture is not used anywhere (at the moment)

Reply:
yes, don't touch the existing path and add the new one by implementing a new class. this new class can reside in VaapiEGL.cpp